generators/vsix-vs-win/BotBuilderVSIX-V4/BotBuilderTemplatesVSIX.csproj

```diff
@@ -1,9 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
+    <MinimumVisualStudioVersion>16.0</MinimumVisualStudioVersion>
     <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
     <ProjectGuid>{2BC10592-C529-4C5E-A9B3-8EF3A830B510}</ProjectGuid>
+    <FileUpgradeFlags>
```

Comment:
This doesn't seem needed and could be removed.

Reply:
Reason is in https://github.com/Microsoft/BotBuilder-Samples/pull/1072/files/06b584b03f132a37357281dbb3f4729ebfb87b8f?utf8=%E2%9C%93&diff=split#r246741310; I'll revert the csproj file. However I do not have VS2017 anymore on my machine.